### DISCUSSION TITLE
Deno.serve API

### DISCUSSION THREAD

**Discussion by rossholdway** [oakserver/oak#613] [Q&A] [answered]
Hi,

Is Oak making use of the performance improvements of Deno.serve now that it is stable, and if not, are there plans to move to using the new Deno.serve API?

https://deno.com/blog/v1.35#denoserve-is-now-stable

Thanks,
Ross

**Comment by kitsonk** [comment, marked as answer]
I am pretty certain the performance improvements were to the stack as a whole and `Deno.serve()` is effectively syntactic sugar for the lower level stack.  When it was originally introduced as unstable, `Deno.serve()` used a seperate HTTP stack, flash, which now has been integrated into the main.

Because of the way oak handles requests, the lower level API is still most appropriate. Therefore for users of oak, there is nothing to due but upgrade the version on Deno they are using.
